bm_scrapy/extensions.py

```diff
@@ -29,9 +20,19 @@ def from_crawler(cls, crawler):
 
     def item_scraped(self, item):
         item = self.exporter.export_item(item)
-        self.producer.send('spider-items', value=dict(item))
+        data = {
+            'jid': os.getenv('BM_JOB'),
```

Comment:
Maybe we should explain BM_JOB in the README.md, and change the name to a someting less general, DB_SPIDER_JOB. Maybe also KAFKA_ADVERTISED_HOST_NAME and KAFKA_ADVERTISED_PORT.

Reply:
Done